nautobot_plugin_chatops_panorama/worker.py

```diff
@@ -107,10 +117,62 @@ def panorama(subcommand, **kwargs):
     return handle_subcommands("panorama", subcommand, **kwargs)
 
 
+@subcommand_of("panorama")
+def get_devices(dispatcher, **kwargs):
+    """Get information about connected devices from Panorama."""
+    pano = connect_panorama()
+    dispatcher.send_markdown(
+        f"Hey {dispatcher.user_mention()}, I'm gathering information about the devices connected to this Panorama as requested.",
+        ephemeral=True,
+    )
+    devices = get_devices_from_pano(connection=pano)
+    dispatcher.send_markdown(
+        f"{dispatcher.user_mention()}, here are the connected devices as requested.", ephemeral=True
+    )
+    dispatcher.send_large_table(
+        ["Hostname", "Serial", "DeviceGroup", "IP Address", "Active", "Model", "OS Version"],
+        [list(device.values()) for _, device in devices.items()],
+        title="Device Inventory",
+    )
+    return CommandStatusChoices.STATUS_SUCCEEDED
+
+
+@subcommand_of("panorama")
+def get_devicegroups(dispatcher, **kwargs):
+    """Get information about DeviceGroups and their associated devices from Panorama."""
+    pano = connect_panorama()
+    dispatcher.send_markdown(
+        f"Hey {dispatcher.user_mention()}, I'm gathering information about the DeviceGroups connected to this Panorama.",
+        ephemeral=True,
+    )
+    devicegroups = get_devicegroups_from_pano(connection=pano)
```

Comment:
based on previous comment this could become `get_from_pano(connection=pano, groups=True, devices=False)`